build.gradle

```diff
@@ -35,6 +43,11 @@ dependencies {
         exclude group: 'org.codehaus.groovy', module: 'groovy-all'
         exclude group: 'org.hamcrest', module: 'hamcrest-core'
     }
+
+    testCompile "org.eclipse.jetty:jetty-server:9.3.6.v20151106"
```

Comment:
Does this indicate that we will be supporting this updated plugin for AEM 6+ only?